index.html

```diff
@@ -479,9 +479,11 @@ <h2>This is an interactive API</h2>
                   let p = curve.get(t);
                   let n = curve.normal(t);
                   let kr = curve.curvature(t);
+                  console.log(Math.abs(kr.k))
```

Comment:
while I'm sure it was useful for debugging, please don't leave console logs in PR code =)

Reply:
Has removed!